lib/manageiq/environment.rb

```diff
@@ -109,7 +109,7 @@ def self.seed_database
 
     def self.setup_test_environment(task_prefix: '', root: APP_ROOT)
       puts "\n== Resetting tests =="
-      run_rake_task("#{task_prefix}test:vmdb:setup", :root => root)
+      run_rake_task("#{task_prefix}test:vmdb:setup", :root => root, :env => {"RAILS_ENV" => "test"})
```

Comment:
I believe we thought we needed this but this was before we realized `verify_no_db_access_loading_rails_environment` has the one polluting the environment, preventing us from setting up the test database correctly.  Can you verify this is still needed?

Reply:
Actually, I understand my question.  I'll check if this is needed.

Reply:
I have tested this manually and it was not needed. Keenan dropped this change.